Description: Javascript dependencies path changes
Last-Update: 2012-11-15
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/icinga-web/inGraph/lib/nodejs/ingraph-renderer.js
+++ b/icinga-web/inGraph/lib/nodejs/ingraph-renderer.js
@@ -10,7 +10,7 @@
 
 var fs = require('fs');
 var src = '';
-['jquery-1.5.min.js',
+['/usr/share/javascript/jquery/jquery.min.js',
 'jquery.flot.node-canvas.js',
 'jquery.flot.text.js'].forEach(function(file) {
     src += fs.readFileSync(__dirname + '/' + file) + "\n";
